### DISCUSSION TITLE
雾凇输入法在新的版本 1.0.1 下，候选词无法横排显示了，为什么？

### DISCUSSION THREAD

**Discussion by tbfan** [rime/home#1576] [General]
雾凇输入法在新的版本 1.0.1 下，候选词无法横排显示了。设置文件中已经设置 
patch/style:   
  text_orientation: horizontal  # horizontal | vertical
  horizontal: true

如何才能设置候选词为横排方式？谢谢！

**Comment by LEOYoon-Tsaw** [comment]
text_orientation 是文字方向，橫排就是你現在看到的100年前才剛誕生的橫排漢字
候選排列方式是candidate_list_layout: linear/stack

**Comment by tbfan** [comment]
的確是这个配置：candidate_list_layout: linear 
谢谢！Rime rocks！
